packages/component-library/draft/Kaizen/Collapsible/styles.scss

```diff
@@ -61,6 +66,8 @@ $heading-active-color: $kz-color-wisteria-100;
 
 .section {
   background-color: white;
+  border-radius: 0 0 $kz-border-solid-border-radius
+    $kz-border-solid-border-radius;
```

Comment:
![image](https://user-images.githubusercontent.com/702885/78868662-a4f5cd00-7a86-11ea-875d-41a95992daa1.png)

Tricky one, but I think this needs to be removed once it becomes sticky. Or if it's sticky mode, maybe it's easier to leave the round borders out for now (like previous). I'm not sure I'm if point at the right line of code here.

Reply:
Back to square borders now: https://dev.cultureamp.design/ally/border-radius/storybook-static/iframe.html?id=collapsible-react--collapsible-group